content/topics/Analyze/causal-inference/did/pretrends.md

```diff
@@ -0,0 +1,178 @@
+
+### Low statistical power
+
+The first major problem with the conventional tests for pre-trends is that they often have low [statistical power](https://tilburgsciencehub.com/topics/analyze/causal-inference/introduction/statistical-power/). Statistical power is a measure of how likely we are to find a statistically significant effect (or difference) if there is one, given the characteristics of our design, such as sample size, the significance level, and the minimum effect size we want to be able to detect. 
+
+Low statistical power essentially means that we will not detect statistically significant differences in pre-trends unless they are very large. This is problematic because small or medium differences can also pose a threat to the parallel trends assumption and thus to reliable causal inference in DiD and event study settings. 
+
+{{% tip %}}
+Read more about statistical power and the problems associated with low statistical power [here](https://tilburgsciencehub.com/topics/analyze/causal-inference/introduction/statistical-power/). 
+{{% /tip %}}
+
+Going back to the He and Wang (2017) paper, let’s imagine that there are differences in trends between the treated and untreated villages before the treatment is introduced. For example, the trend in poor housing is already decreasing in the treated villages, while in untreated ones it is not. However, the test we use to check for parallel trends has lower power and we are unable to detect this difference, so we assume that the parallel trends assumption holds and proceed to estimate the treatment effect. We find that the difference in differences is negative, that is, the treatment has apparently decreased the share of poor housing. Since the trend before the treatment was introduced was already like this, we cannot attribute this observed difference to the treatment, but we do not know this because low statistical power impedes the detection of the pre-treatment difference in trends. In this way, using low-powered pre-trend tests to verify the parallel trends assumption can lead to mistaken causal inference. 
+
+Low statistical power is not a problem exclusive to tests for pre-trends: it also often occurs with the estimation of the treatment effects of interest. However, it can occur more frequently with pre-trend tests due to lower sample sizes (if we are looking at each pre-treatment period separately) and higher levels of noise in the outcome before treatment is implemented. In any case, it is always useful to assess the power our pre-trends test has to evaluate its capacity to detect significant differences in pre-trends and establish the credibility of the parallel trends assumption. 

```

Comment:
this should be shortened as well, maybe stick to the main point? For example, shorten to:

Low statistical power is especially likely to happen in pre-trend tests due to a smaller sample size and higher levels of noise in the outcome before the treatment is implement. It is useful to assess the power to evaluate the capacity of our pre-trend tests.